src/app/shared/components/dialog/dialog.component.html

```diff
@@ -0,0 +1,8 @@
+<h1>{{data.title}}</h1>
+<div>
+  <label id="favorite-animal" [innerHTML]="data.question"></label>
```

Comment:
interesting id, I think you can remove it

Reply:
hahaha)